recorded_future/plugin.spec.yaml

```
@@ -68,7 +68,7 @@ types:
     type:
       type: string
       required: false
-  evidenceDetails:
+  evidenceDetails_obj:
```

Any reason we changed this name vs. keeping it the same?

```suggestion
  evidenceDetails:
```

---

She was avoiding the bug where if you have an object in an object of the same name you get a regen error.

---

@jmcadams-r7 this doesn't hit the bug though? If the parent type is a different type than the child types it contains that should not cause that scenario to happen. What bug was being encountered? Any logs?